README_Sink.md

```diff
@@ -0,0 +1,182 @@
+# Kafka Connect Sink Connector for Azure IoT Hub
+________________________
+
+The Sink connector allows you to send messages to Azure IoT devices by simply posting messages to Kafka topics. The
+connector sends the messages to Azure IoT Hub, which in turn forwards them to the right devices. The
+messages need to be in a specific format (details below), that allow the connector to extract the information
+necessary to send them to the right devices.
+
+### Configuration
+
+All the configuration settings required to run Kafka sink connector for IoT Hub are in the
+"[connect-iothub-sink.properties](connect-iothub-sink.properties)" file. The properties related to Azure IoT Hub
+can be obtained from the [Azure Portal](https://portal.azure.com). For more information on how to get the IoT Hub
+settings, please refer to the documentation
+[here](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-java-java-getstarted) and
+[here](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-java-java-c2d).
+
+Here are the configurable values -
+
+| Config | Value Type | Required | Default Value | Description |
+|-------------|-------------|-------------|-------------|----------------|
+| topics | String | Yes |  | The list of topics from which messages will be retrieved to send to IoT devices. |
+| tasks.max | Int | Yes |  | Maximum number of tasks that should be created for this connector. |
+| IotHub.ConnectionString | String | Yes |  | The IoT Hub connection string. In the Azure Portal, you can find the value under to "IoT Hub" >> your hub >> "Shared access policies" >> "service" >> "Connection string" |
+| IotHub.MessageDeliveryAcknowledgement | String | No | None | The type of delivery acknowledgement you want for the messages sent to the devices. The valid values are None, Full, PositiveOnly and NegativeOnly.|
+
+Here is a sample connect-iothub-sink.properties file. (Some of these values are not configurable, and hence omitted from
+ the list above)
+
+```
+connector.class=com.microsoft.azure.iot.kafka.connect.sink.IotHubSinkConnector
+name=AzureIotHubSinkConnector
+tasks.max=1
+topics=testtopic
+IotHub.ConnectionString=HostName=Test.azure-devices.net;SharedAccessKeyName=service;SharedAccessKey=aBCdeBb5HffTTs/J9ikdcqab1JNMB0ot=
+IotHub.MessageDeliveryAcknowledgement=None
+```
+
+> Note: At the moment, the Sink connector does not support getting feedback from the devices on the status of the
+> messages (whether accepted or rejected). If you want to get feedback, you will need to do it manually. Please refer
+> to the documentation [here](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-java-java-c2d) on how to do that.
+
+
+### Data format
+
+The sink connector expects the messages in the Kafka topic to have the schema below, so that it can extract the
+information required to send the message to the IoT device. If a record is not in the expected format, the connector
+will throw an exception.
```

Comment:
perhaps we should clarify if the connector stops or continues to work, ignoring the poison message